controller/skillreviewscontroller.go

```diff
@@ -83,6 +87,45 @@ func (c *SkillReviewsController) getAllSkillReviews() error {
 	return err
 }
 
+func (c *SkillReviewsController) getReviewForSkill(skill_id string) error {
+	opts := data.NewCassandraQueryOptions("skill_id", "'" + skill_id + "'", true)
```

Comment:
Try this with false as the Id option.  If that doesn't work, I'll put this through and make a story about handling this kind of request